src/tests/unit/tests/background/telemetry/app-insights-telemetry-client.test.ts

```diff
@@ -1,24 +1,24 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { IConfig, ITelemetryContext, ITelemetryTrace } from '@microsoft/applicationinsights-common';
```

Comment:
You can avoid the need for the extra `@microsoft/applicationinsights-common` dependency by importing `IConfig` from the re-export in `@microsoft/applicationinsights-web` and replacing the `ITelemetry*` imports with:

```ts
    type ITelemetryContext = Initialization["context"];
    type ITelemetryTrace = ITelemetryContext["telemetryTrace"];
```

Reply:
ooh nice catch!!